src/uploads/deleteAssociatedFiles.ts

```diff
@@ -31,26 +30,28 @@ export const deleteAssociatedFiles: (args: Args) => Promise<void> = async ({
 
     const fileToDelete = `${staticPath}/${doc.filename}`;
 
-    if (await fileExists(fileToDelete)) {
```

Comment:
In this case, it might be correct. For consistency in the whole file, I chose to adjust both.

An alternate approach would be to change to a for loop instead of `forEach` and change it back to await syntax.

Reply:
Do you want me to change it?

Reply:
Yep, that'd be great!